projects/addon-charts/components/line-days-chart/line-days-chart-hint.directive.ts

```diff
@@ -62,15 +62,17 @@ export class TuiLineDaysChartHint implements AfterContentInit {
         return this.getMap(...this.charts.map(({value}) => value)).get(String(day)) || [];
     }
 
-    public raise(day: TuiDay): void {
+    public raise(day?: TuiDay): void {
```

Comment:
Let's not make it optional and instead add early return where it is called. Same with onHover there

Reply:
fixed